tests/test_actions.py

```diff
@@ -185,10 +185,10 @@ def test_get_operator_no_phone(
 @pytest.mark.parametrize(
     "language, date, expected_events",
     [
-        ("de-DE", "2023-04-24",
-         [BotUttered('{"motd": {"language": "de-DE", '
-                     '"motd": ["Hallo! Montag ist ein frischer Start.", '
-                     '"Womit kann ich Ihnen heute behilflich sein?"]}}')]),
+        ("is-IS", "2023-04-24",
+         [BotUttered('{"motd": {"language": "is-IS", '
+                     '"motd": ["H\\u00e6! M\\u00e1nudagur markar n\\u00fdtt upphaf.", '
+                     '"Hva\\u00f0 get \\u00e9g hj\\u00e1lpa\\u00f0 \\u00fe\\u00e9r me\\u00f0 \\u00ed dag?"]}}')]),
```

Comment:
I've given up on solving this Unicode problem in a neater way. It's really not a trivial matter to try and encode/decode these messages so that they line up when one is returned through Rasa. If anyone has any ideas on improving this, I'm all ears. But otherwise we can maybe just leave it like so?

Reply:
Fix from @bnika in latest commit.